backend/src/utils/handler_decorators.py

```
@@ -0,0 +1,290 @@
+            return {"message": "success"}
+    """
+    def decorator(func: Callable) -> Callable:
+        decorated_func = func
+        
+        # Apply decorators in reverse order (innermost first)
+        if handle_errors:
+            decorated_func = standard_error_handling(decorated_func)
+        
+        if require_ownership:
+            if not isinstance(require_ownership, tuple) or len(require_ownership) != 2:
+                raise ValueError("require_ownership must be a tuple of (resource_param, resource_type)")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)